(chore) add missing expo useoauth hook to sidenav (#1797)
clerk/clerk-docs: docs/references/expo/use-local-credentials.mdx

The `useLocalCredentials()` hook enables you to store a user's password credentials on their device and subsequently use biometrics for sign-in. This enhances the user experience by allowing users to sign in using Face ID or another biometric authentication method during future sign-ins.

> [!WARNING]

> > This hook is not supported in web apps and can only be used in native apps.

Reviewer: ```suggestion
> This hook isn't supported in web apps and can only be used in native apps.
```
